src/rs_quark_assets/components/A2HSBanner.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <div v-if="eventTriggered !== 'triggered'">
+    <transition
+      enter-active-class="ease-out duration-300 motion-safe:transition"
+      enter-from-class="translate-y-2 opacity-0 sm:translate-y-0 sm:translate-x-2"
+      enter-to-class="translate-y-0 opacity-100 sm:translate-x-0"
+      leave-active-class="motion-safe:transition ease-in duration-100"
+      leave-from-class="opacity-100"
+      leave-to-class="opacity-0"
+    >
+      <div>
+        <div
+          v-if="installPromptEvent"
+          aria-live="assertive"
+          class="installPrompt-center max-w-sm overflow-hidden rounded-lg bg-white shadow-lg ring-1 ring-black ring-opacity-5"
+        >
+          <div class="p-4">
+            <div class="flex items-center gap-4">
+              <div
+                class="rounded-full border-2 border-green-400 p-2"
+                @click="useA2hs()"
+              >
+                <SvgCheck class="cursor-pointer text-green-400" />
+              </div>
+              <div @click="useA2hs()">
+                <p class="text-lg font-medium text-muted">
+                  {{ messages.installPrompt }}
+                </p>
+              </div>
+              <button
+                class="inline-flex"
+                @click="emit('triggered', 'triggered')"
+              >
+                <span class="sr-only">Close</span>
+                <SvgX
+                  class="cursor-pointer text-blue-500 hover:text-blue-700"
+                />
+              </button>
+            </div>
+          </div>
+        </div>
+        <div
+          v-else-if="isIosSafari"
+          aria-live="assertive"
+          class="ios-center max-w-sm overflow-hidden rounded-lg bg-white shadow-lg ring-1 ring-black ring-opacity-5"
+        >
+          <div class="p-4">
+            <div class="flex items-center gap-4">
+              <div class="rounded-full border-2 border-green-400 p-2">
+                <SvgCheck class="cursor-pointer text-green-400" />
+              </div>
+              <div>
+                <p class="text-lg font-medium text-muted">
+                  {{ messages.ios }}
+                </p>
+              </div>
+              <button
+                class="inline-flex"
+                @click="emit('triggered', 'triggered')"
```

Comment:
```suggestion
                @click="emit('triggered')"
```
nitpick: the triggered Function has no argument